lib/model/QubitPhysicalObject.php

```diff
@@ -152,25 +152,41 @@ public static function getPhysicalObjectsByNameAndLocation($name, $location, $ty
    */
   public static function checkPhysicalObjectExistsInCollection($name, $location, $typeId, $collectionId)
   {
+    if (!isset($collectionId))
+    {
+      return;
+    }
+
+    // Get collection lft and rgt values from the database as they are not
+    // always being updated in the class cache in multi-level imports
+    $sql = "SELECT lft, rgt FROM information_object WHERE id = :id;";
```

Comment:
I'm not completely sure if this is needed with double quotes because of the params, I'll double check ;)